.licignore

```diff
@@ -5,9 +5,9 @@
 !Makefile
 !*.go
 !Dockerfile*
+!hack/boilerplate.gotxt
```

Comment:
viz my previous comment, we can keep original filename `!hack/boilerplate.go.txt`

Reply:
@kuritka , this will affect all *.txt files in the project, I need go style comment to be applied, not plain text, as `boilerplate.go.txt` is going to be used by controller-gen and will be injected to [api/v1beta1/zz_generated.deepcopy.go](https://github.com/AbsaOSS/k8gb/pull/375/files#diff-ea922c6dc50e1006dbedec93a5d58173a622630990e2985d97d6c088cf9cce79)